tests/periph_spi/tests/04__periph_spi_timer.robot

```diff
@@ -0,0 +1,60 @@
+*** Settings ***
+Documentation       Verify that the meassured times are within a certain window
+
+Suite Setup         Run Keywords    PHILIP Reset
+...                                 RIOT Reset
+...                                 API Sync Shell
+...                                 API Firmware Should Match
+Test Setup          Run Keywords    PHILIP Reset
+...                                 RIOT Reset
+...                                 API Sync Shell
+...                                 SPI Init Should Succeed
+Test Teardown       Run Keywords    SPI Release Should Succeed
+
+Resource            periph_spi.keywords.txt
+Resource            api_shell.keywords.txt
+
+Variables           test_vars.py
+
+Force Tags          periph  spi
+
+
+*** Keywords ***
```

Comment:
This keyword is very specific for this test and will probably never be used somewhere else since it is pretty much the whole routine for every test case in this file to avoid code duplication. In my opinion the code is more readable if it stays there. But if you want it in the keywords file nevertheless I can move it there.

Reply:
Sounds good leave it as is!